lms/djangoapps/bulk_email/tasks.py

```diff
@@ -195,6 +196,12 @@ def perform_delegate_email_batches(entry_id, course_id, task_input, action_name)
     if total_recipients <= settings.BULK_EMAIL_JOB_SIZE_THRESHOLD:
         routing_key = settings.BULK_EMAIL_ROUTING_KEY_SMALL_JOBS
 
+    # Weird things happen if we allow empty querysets as input to emailnig subtasks
```

Comment:
"weird things" are all related to the task progress and history displays - they get all out of sorts and never finish is the task is at state `Completed 0 of 0 subtasks`

This method of handling it will result in the task being queued successfully, and these results when inspected later:
<img width="1131" alt="screen shot 2016-06-02 at 12 36 30 pm" src="https://cloud.githubusercontent.com/assets/7373924/15752988/abfb07aa-28be-11e6-884d-05dd14e82b86.png">